tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun/pipelinelooprun.go

```diff
@@ -654,36 +675,31 @@ func (c *Reconciler) updatePipelineRunStatus(logger *zap.SugaredLogger, run *v1a
 				}
 			}
 		}
-		for _, runStatus := range pr.Status.Runs {
```

Comment:
I think we should also check with pr.Status.Runs as well because we might move the break task into a custom task to further optimize our pipelines.